controller/service.rst

```diff
@@ -23,17 +23,19 @@ These are the main **advantages** of defining controllers as services:
   service container configuration. This is useful when developing reusable bundles;
 * Your controllers are more "sandboxed". By looking at the constructor arguments,
   it's easy to see what types of things this controller may or may not do;
+* If you're not passing some required dependencies or if you are injecting some
```

Comment:
I think it is worth clarifying the semantic above, because technically this is not the constructor of a controller but of a class of controllers, in this way a Dunglas action is more a controller than the base controller class.